diagnostic.md

```diff
@@ -94,4 +94,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+'teamMember['hobby']' = 'exercise', `teamMember['name']` = 'Jason' 'Weeks', `teamMember['name']['surname']` = 'Weeks'
```

Comment:
Very close. teamMember['name'] would actually return the entire object, like this: {given: 'Jason', surname: 'Weeks'}. 
Computers have no ability to interpret our prompts, so it would not be able to combine value of 'given' and 'surname' without explicit direction to do so.